README.md

```diff
@@ -82,9 +82,17 @@ The sonoff-ota-flash.sh script has been tested with the following operating syst
 
 | OS                   | Working | Notes                        |
 |----------------------|---------|------------------------------|
-| Mac OS 12.3.1        |   ✅    | dns-sd/1558.0.56 curl/7.79.1 |
-| Raspberry Pi OS 11.3 |   ✅    | avahi-browse/0.8 curl/7.74.0 | 
-| Windows 10           |   ❌    | Likely problems with Multicast DNS for device discovery? |
+| Mac OS 12.3.1        |    ✅   | dns-sd/1558.0.56 curl/7.79.1 |
+| Raspberry Pi OS 11.3 |    ✅   | avahi-browse/0.8 curl/7.74.0 | 
+| Windows 10 and 11    |    ✅   | Need to do the following Windows requirements** | 
+
+**Windows 10 and 11 requirements: Need to install Windows Subsystem for Linux, then install Debian from the Windows Store, open Debian Linux command prompt and run the following commands:
+```
+sudo apt-get update
+sudo apt-get install curl avahi-utils
```

Comment:
Why install avahi-utils, if it doesn't work?

Reply:
It didn't work on my setup, but didn't know if it might work for others?  After removing avahi-utils, of course I get an error when running the script without -i, so anyone else that might want to test multicast DNS discovery on their setup won't be able to without installing avahi-utils.  Installing that component would allow your script to run without the "didn't find dns-sd or avahi-browse" error. :) I have a ton more of these Sonoff MiniR2 to flash, so I'll test the script more without avahi-utils installed while using the -i option.

Reply:
For anyone else like me using Linux on Windows (WSL), after installing avahi-utils, you also have to run the following commands to start the needed services:
sudo service dbus start
sudo service avahi-daemon start

However, I still get the following error when running the script and the device can't be automatically found.
Searching for Sonoff module on network...
Failed to find a Sonoff module on the local network.

I removed avahi-utils, ran the script with -i to specify the device IP and the script works perfectly.  So, line 92 could be modified to remove avahi-utils if you'd like since the -i option doesn't need it and script runs without error when using that option.  Thanks for all your hard work creating a very helpful script.  BTW, I used your script because the Sonoff provided tool fails to work correctly now on Windows 10/11, even after installing QT to get their app to run without error.